docker/adv-mon-stack/src/adv-monstack.py

```diff
@@ -27,15 +27,19 @@ def request(request_method,Api_Token,Api_Url,info,files,name):
 
```

Comment:
Add minimal documentation using comments, maybe for each function, what is happening.  In readme, the purpose, overview of what its doing